script.js

```diff
@@ -1,4 +1,294 @@
+// beat-that game base + score + leaderboard + 2mode (normal & lowest combined number) + auto generate combined number
+// 2 players and players take turns
+
+// 4 game modes: "player 1 dice roll", "player 1 choose dice order", "player 2 dice roll", "player 2 choose dice order", and the game restart
+
+// 2 mode: "normal" (highest combined number win) || "reverse" (lowest combined number win)
+
+var currentGameMode = "waiting for user to pick the mode";
+var mode;
+var currentPlayer = 1;
+var player1CombineNumber = 0;
+var player2CombineNumber = 0;
+var player1TotalScores = 0;
+var player2TotalScores = 0;
```

Comment:
Whenever we have more than 1 player in a game, it's usually good to store the values in a data type that can hold multiple values (Just in case there will e multiple players next time) 

So my variables might look something like this:
Can either use an array or object 
```
// Array method 
// index 0 is playerOne and index 1 is player 2 
var playersTotalScore= [10,8]

//Object method
var playersTotalScore= {
    1: 10,
    2: 8
}
```